### ISSUE TITLE
Composer Support

### ISSUE THREAD

**Issue by browner12** [fusioncharts/php-wrapper#16]
There is an issue (#6) from 2019 requesting Composer support, which was never implemented.

Could we please revisit this super low hanging fruit request?

It appears you guys already have a Packagist account (https://packagist.org/packages/fusioncharts/), so it should be really simple to add this repo as well.

Thanks for the consideration.

**Comment by AyanBhadury** [comment]
@browner12 You can download fusionexport PHP SDK from composer : 
`composer require fusioncharts/fusionexport`

Regarding fusioncharts PHP wrapper it's a simple utility file which on adding will help you to render charts from the PHP framework.

**Comment by renoirtech** [comment]
@AyanBhadury yeah, but it's awful to maintain on Laravel or any other composer project. Please add composer support and an "updated" way of using Fusioncharts on Laravel Projects, please!

**Comment by browner12** [comment]
clearly there's no interest in this so closing.
